src/clients/inst_setup_dhcp.rb

```diff
@@ -1,12 +1,24 @@
 require "yast"
 require "network/network_autoconfiguration"
 
-def main
-  Yast::NetworkAutoconfiguration.instance.configure_dhcp
+module Yast
+  class SetupDhcp
```

Comment:
Could be. These (networking) clients were created in early times of rewriting installer when the structure was not fixed. 

As I should move more clients, I'd leave it for SP3.